web_src/js/components/ViewFileTreeItem.vue

```diff
@@ -101,7 +103,7 @@ const doGotoSubModule = () => {
     </div>
```

Comment:
![image](https://github.com/user-attachments/assets/290eba64-7d00-4578-b963-cdaaf02e6bed)

Maybe also this one?

Reply:
This line is not needed, because svg under item-toggle has no span wrap, and there is no problem with style.

Reply:
I mean, the name is not right.

That name was designed for "actions job status"

If you need to use it somewhere else, then it needs a new general name

Reply:
See the "FIXME" above